xapian-applications/omega/omegatest

```diff
@@ -414,6 +414,20 @@ printf '$highlight{$cgi{text},$cgi{list},$cgi{open},$cgi{close}}' > "$TEST_TEMPL
 testcase 'A list of <b>words</b>' list="words" text="A list of words" open="<b>" close="</b>"
 testcase 'A *list* of *words*' list="words${tab}list" text="A list of words" open="*" close="*"
 
+#Test setting seterror and mset size, should not run the query after setting error.
```

Comment:
Please put a space between the `#` and the comment text for readability (and also consistency with all the existing comments in this file and elsewhere in the codebase).  Also below.